### ISSUE TITLE
x^y not implemented?

### ISSUE THREAD

**Issue by akalongman** [hoaproject/Math#25]
x^y functionality not implemented?

**Comment by Hywan** [comment]
Nop. But [`pow`](http://php.net/pow) can do the trick. Are you talking about the arithmetical grammar?

**Comment by akalongman** [comment]
Yes, but in math expressions pow synonim is ^. Can you implement this functionality too?

**Comment by Hywan** [comment]
Yes, sure.

/cc @hoaproject/hoackers that's an easy PR, any volunteers?

**Comment by osaris** [comment]
It means adding an alias in https://github.com/hoaproject/Math/blob/7d159567f4ee10de7391f5042dc957fd041f9dec/Visitor/Arithmetic.php#L343 to call pow when ```^``` is used ?

**Comment by akalongman** [comment]
Yes, but format is other, script must parse `x^z` as `pow(x, z)`

**Comment by osaris** [comment]
Label should not be "in progress" ?
